src/components/stacked-area-chart/StackedAreaChart.scss

```diff
@@ -0,0 +1,9 @@
+.stacked-area-chart {
+    width: 100%;
+    height: 100%;
+}
+
+.chart-data {
```

Comment:
Duplicate css class from another file, if i remember correct, we can prefix it with `stacked-area`.